README.md

```diff
@@ -3,68 +3,184 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 ## Available Scripts
+<div align="center">
 
-In the project directory, you can run:
+  
+  <br/>
 
-### `npm start`
+  <h3><b>CRYPTO</b></h3>
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+</div>
```

Comment:
- [ ] You should put links for your presentation and deployed app in a readme file for more visibility and it is requested. Also remember to highlight in vscode a piece of code your find interesting or you are proud of making.

Reply:
well resolved! ✔️